modules/datastore.js

```diff
@@ -0,0 +1,35 @@
+// Imports the Google Cloud client library
+const Datastore = require('@google-cloud/datastore');
```

Comment:
tested this locally (by running `node modules/datastore.js`) and got an error that Datastore is not a constructor. i did some research and the fix is to change the line like so:
```suggestion
const { Datastore } = require('@google-cloud/datastore');
```

other than that, lgtm!

Reply:
Thank you. Quick change no problem